stable/syncthing/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "syncthing.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app.kubernetes.io/name: {{ include "syncthing.name" . }}
```

Comment:
these label names differ from the other ones used.
There was a switch in the best-practice naming - but please use a consistent set throughout the chart.

Reply:
Fixed.